rules/windows/file_rename/file_rename_win_not_dll_to_dll.yml

```diff
@@ -15,10 +15,10 @@ detection:
     to_dll:
         TargetFilename|endswith: '.dll'
     filter_from_dll:
-        - OriginalFilename|endswith:
```

Comment:
From mapping for file_rename: 
`# Aurora Agent adds Image, CommandLine, Hashes, ParentImage, User, OriginalFilename, TargetFilename`

Reply:
This has been adjusted in Aurora. Sysmon uses `OriginalFileName`.